apps/vela-api/test/routes/dictionary.test.ts

```
@@ -0,0 +1,212 @@
+            senses: [{ english_definitions: ['Tokyo'] }],
+            jlpt: [],
+            is_common: true,
+          },
+        ],
+      }),
+    });
+
+    const app = createTestApp();
+    await app.request('/lookup?word=東京', { headers: AUTH_HEADER });
+
+    const fetchInit = mockFetch.mock.calls[0]?.[1] as RequestInit | undefined;
```

ESLint

[error] 'RequestInit' is not defined.

(no-undef)